_posts/2019-07-19-create-device-groups.adoc

```diff
@@ -0,0 +1,66 @@
+= Group your devices
+:page-layout: page
+:page-categories: [usage]
+:page-date: 2019-07-19 10:22:16
+:page-order: 2
+:icons: font
+
+Once enough devices have been provisioned, you should organize them into groups so that you can target them in your campaigns.
+
+There are two types of group: *fixed group* and *smart group*. 
+
+* The *fixed group* type is useful if you already have a fixed list of devices that you want to update.
+* The *smart group* type is useful if you know the selection criteria for devices to update but don't yet have a fixed list of devices. 
++
+Even if you do have a list, smart groups can also come in handy when your list is constantly being appended with new devices. Smart groups automatically group newly provisioned devices that match certain selection criteria.
+
+== Create Fixed Groups
+
+*To create a fixed group, follow these steps:*
+
+1.  Open the https://connect.ota.here.com/#/devices[Devices] tab and click *+Add group*.
+1.  Select *Fixed Group* and click Next.
+1.  In the next window, enter a group name.
+1.  Assign devices to your group.
```

Comment:
This isn't correct. When creating Fixed groups, you go through the wizard completely and create an empty group. Then you need to drag and drop devices into it. There is no selection of devices during the group creation.